src/components/elements/table/GenericTable.tsx

```diff
@@ -80,24 +81,16 @@ const clickableRowStyles = {
   cursor: 'pointer',
 };
 
-const HeaderCell = ({
-  children,
-  sx,
```

Comment:
Not really part of this diff, but maybe we can get rid of `headerCellSx` as we are standardizing tables - started thread https://www.figma.com/design/bsw8CX4UOq6SIdXYorZDWV?node-id=2424-868&m=dev#1081881485